src/servers/esl.js

```diff
@@ -13,15 +13,19 @@ class EslServer extends SocketServer {
   }
 
   holdCall(callId) {
+    this._getSocket(callId).event_json('CHANNEL_HANGUP');
+    this._getSocket(callId).filter('variable_conference_name', callId);
```

Comment:
I understand that FS prefixes with _variable__. My point is: your variable is _conference_name_ here, but it is _conference_ in the next line:
```
return this._getSocket(callId).execute('conference', callId + '@video-mcu-stereo++flags{mintwo}')
```
and in the _bridgeCall()_ method.

This seems suspicious to me, though I might not be getting sth here :)